jerry-core/include/jerryscript-core.h

```diff
@@ -90,6 +90,11 @@ typedef enum
   JERRY_FEATURE_SNAPSHOT_EXEC, /**< executing snapshot files */
   JERRY_FEATURE_DEBUGGER, /**< debugging */
   JERRY_FEATURE_VM_EXEC_STOP, /**< stopping ECMAScript execution */
+  JERRY_FEATURE_JSON, /**< JSON support */
+  JERRY_FEATURE_PROMISE, /**< promise support */
+  JERRY_FEATURE_TYPEDARRAY, /**< Typedarray support */
+  JERRY_FEATURE_DATE, /**< Date support */
+  JERRY_FEATURE_REGEX, /**< Regex support */
```

Comment:
The standard calls it `RegExp` and not `Regex`, so please rename to be consistent. http://www.ecma-international.org/ecma-262/5.1/index.html#sec-15.10
`JERRY_FEATURE_REGEXP, /**< RegExp support */`